site/en/docs/extensions/mv3/service_workers/index.md

```diff
@@ -96,9 +97,13 @@ chrome.bookmarks.onCreated.addListener(() => {
   // do something
 });
 ```
+{% endCompare %}
 
-Do not register listeners asynchronously, as they will not be properly triggered.
+If you register the same listener asynchronously (for example, inside a `runtime.onInstalled` listener), 
+the event will be dropped because the nested listener will not be registered when the browser 
+attempts to dispatch it.
 
+{% Compare 'worse' %}
 ```js
 chrome.runtime.onInstalled.addListener(() => {
   // ERROR! Events must be registered synchronously from the start of
```

Comment:
Ah! Almost missed this. We should update the terminology in this comment to reflect the change made to the second paragraph after the "Set up listeners" header.